index.html

```diff
@@ -1691,7 +1691,72 @@ <h3>Add Proof</h3>
         </p>
 
       </section>
+      <section>
+        <h3>Add Proof Set</h3>
+        <p>
+          The following algorithm specifies how to add a proof set to a 
+          document. Required inputs are an
+          <a>unsecured data document</a> (<var>unsecuredDocument</var>) and a
+          (unordered) list of
+          <a>proof options</a> (<var>options_list</var>). Each <a>proof options</a>
+          (<var>options</var>) in the list must satisfy the criteria in 
+          section <a href="#add-proof"></a>.
+          A <dfn>secured data document</dfn> is produced as
+          output. Whenever this algorithm encodes strings, it MUST use UTF-8 encoding.
+        </p>
 
+        <ol class="algorithm">
+          <li>
+Let <var>output</var> be a copy of <var>unsecuredDocument</var> and 
+<var>proof_list</var> be an empty list.
+          </li>
+          <li>
+For each <var>options</var> in <var>options_list</var> run steps 2 through 8 of
+the algorithm in section <a href="#add-proof"></a>. If no exceptions are raised 
+append the generated <var>proof</var> value to the <var>proof_list</var> 
+otherwise raise the exception.
```

Comment:
```suggestion
For each <var>option</var> in <var>options_list</var>, run steps 2 through 8 of
the algorithm in section <a href="#add-proof"></a>. If no exceptions are raised,
append the generated <var>proof</var> value to the <var>proof_list</var>;
otherwise, raise the exception(s).
```

Reply:
Same naming issue as previously mentions each **proof** has a set of **proof options**.

Reply:
I'm going to try rewriting the algorithms in terms of incremental additions to proof sets/chains rather than processing of any array of proof options. This should help on the variable naming and should be closer to the typical case of adding signatures individual as a VC flows from signer to signer.

Cheers Greg

Reply:
`each <var>option</var> in <var>options_list</var>,` represents no problem of plural/singular agreement. A number of singular items (each being an `option`) are collected together into a list of options, an `options_list`.

Reply:
See simplified update of the algorithm that I just committed. Thanks.